server/app.py

```diff
@@ -0,0 +1,17 @@
+# app.py
+from flask import Flask, render_template, request, url_for, send_from_directory
+import speak
+
+app = Flask(__name__, static_folder="../static/build/static", template_folder="../static/build")
+@app.route("/", methods=["GET","POST"])
+def index():
+    return render_template("index.html")
+
+@app.route("/text/<text>/<lang>")
+def text(text,lang):
+    if request.method == "GET":
+       mp3_path = speak.get_speak(text,lang)
+       return mp3_path
```

Comment:
Should be returning  it with `url_for`  another issue with `random_tts_filename()` function it's joining the static folder path to the TTS filename. The right solution here is what i mentioned before separating the constants, importing and using them wherever needed. But a quick hard-coding will be to remove the absolute path with relative one:
`return url_for('static', filename=mp3_path.replace('../static/', ''))`